README.md

```diff
@@ -63,3 +63,34 @@ After modify the fullnode variables, overwrite the existing setting:
 ```bash
 $ ansible-playbook iri.yml -e overwrite=yes
 ```
+
+To change all service status:
+
+```bash
+# restart service
+$ ansible-playbook manage.yml -e systemd_status=restarted
+# start service
+$ ansible-playbook manage.yml -e systemd_status=started
+# stop service
+$ ansible-playbook manage.yml -e systemd_status=stopped
+```
+
+To restart service by group:
+
+```bash
+# restart broker
+$ ansible-playbook manage.yml --limit broker -e systemd_status=restarted
+# restart fullnode
+$ ansible-playbook manage.yml --limit fullnode -e systemd_status=restarted
+# restart remote-worker
+$ ansible-playbook manage.yml --limit remote-worker -e systemd_status=restarted
+```
+
+To restart service by host:
+
+```bash
+# restart node
+$ ansible-playbook manage.yml --limit node -e systemd_status=restarted
+# restart remote-worker-1
```

Comment:
Why suffix `-1`?

Reply:
In the `inventory`, we define a `remote-worker` group and that includes 4 remote-worker, `remote-worker-1` is one of them.

```
[remote-worker]
remote-worker-1 ansible_host=192.168.0.101 ansible_user=ubuntu
remote-worker-2 ansible_host=192.168.0.102 ansible_user=ubuntu
remote-worker-3 ansible_host=192.168.0.103 ansible_user=ubuntu
remote-worker-4 ansible_host=192.168.0.104 ansible_user=ubuntu
```

Reply:
Since it is a matter of the range of available workers, the implication should be denoted.

Reply:
I think these can be simplified.

---

For example:

To change all service status:

```bash
$ ansible-playbook manage.yml -e systemd_status=[status]
```
The acceptable **[status]** are
- `started`
- `restarted`
- `stopped`

---

The others can be modified as well.

Reply:
I removed this section, and make it simplified as @marktwtn suggestion.